content-src/actions/action-manager.js

```diff
@@ -64,8 +67,8 @@ function RequestBookmarks() {
   return RequestExpect("RECENT_BOOKMARKS_REQUEST", "RECENT_BOOKMARKS_RESPONSE");
 }
 
-function RequestRecentLinks() {
-  return RequestExpect("RECENT_LINKS_REQUEST", "RECENT_LINKS_RESPONSE");
+function RequestRecentLinks(data) {
+  return RequestExpect("RECENT_LINKS_REQUEST", "RECENT_LINKS_RESPONSE", {data: data});
```

Comment:
> So... let's just use `data`!

@k88hudson Do we need to update the [README](https://github.com/mozilla/activity-streams#conventions-for-actions) if we're getting rid of `query`?